nuclio/export.py

```diff
@@ -156,6 +156,23 @@ def from_notebook_node(self, nb, resources=None, **kw):
                     function_buffers[current_name][ended] = False
                     seen_function_name = seen_function_name or current_name
 
+            lines = code.splitlines()
+            if cell_magic in code:
+                self.handle_cell_magic(config,
+                                       lines,
+                                       function_buffers,
+                                       ended,
+                                       code_cells)
+                continue
+
+            if line_magic in code:
+                self.handle_line_magic(config,
+                                       lines,
+                                       function_buffers,
+                                       ended,
+                                       code_cells)
+                continue
```

Comment:
From the existence of the continue in both cases I understand (maybe wrongly ?) that when there is usage of some magic command there can't be any code in the cell (i.e. it won't be "collected" as code)
But looking on `handle_cell_magic` and `handle_line_magic` I see that they are dealing with the function buffers, what am I missing ?

Reply:
there might be code, if there is code it needs to be treated differently so it is handled inside `handle_cell_magic` and `handle_line_magic` so the continue is to avoid duplication of code. this was designed similarly to the original implementation.
the whole point of this is to not do things twice, not adding code and not executing magic commands when it was already done